workspace/workflows/jobs.py

```diff
@@ -189,13 +200,17 @@ def parse_args():
 
 
 def main(org, repo, branch):
-    if org not in config.REPOS.keys():
+    if org == "all":
+        # Summarise status for all repos in all orgs
```

Comment:
Tiny micro-nit: can we make the spelling consistent for "summarise/ize"

Reply:
Yes! I thought I had unified them but probably my brain did not work

Reply:
Actually just realised that there is `summarise_ncso_concessions` so I changed everything to "summarise"